docker/Dockerfile

```diff
@@ -1,22 +1,18 @@
-FROM php:7.1-cli
+FROM php:7.2-cli
```

Comment:
Is this going to build the latest PHP version for the `Docker` environment?

Reply:
It's a prebuilt docker image, so yes it will use the said version.